_includes/partials/callouts.html

```diff
@@ -0,0 +1,23 @@
+<div class="callout">
+  <a class="card1" href="https://bit.ly/2SbrRhe">
+    <h3>Community Calendar</h3>
+    <p class="small">Join the weekly meetings by subscribing to our community calendar</p>
+    <div class="go-corner" href="https://bit.ly/2SbrRhe"></div>
+  </a>
+   <a class="card1" href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA">
```

Comment:
YouTube links need to include the subscription confirmation. 

```suggestion
   <a class="card1" href="https://www.youtube.com/c/Layer5io?sub_confirmation=1">
```